### PR TITLE
Update how-to-enable.md

### PR DESCRIPTION
**Pull request by rededwin**
Add link to Intune advanced deployment guide (FastTrack) that's available in M365 Admin Center

### CODE REVIEW

File: memdocs/configmgr/comanage/how-to-enable.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ The phrase **Pilot group** is used throughout the co-management feature and conf
 
 Make sure the co-management prerequisites are set up before you start this process. For more information, see [Prerequisites](overview.md#prerequisites).
 
+> [!TIP]
+> As a companion to this article, we recommend using the [Microsoft Intune Configuration Manager co-management setup guide](https://go.microsoft.com/fwlink/?linkid=2224782) when signed in to the Microsoft 365 Admin Center. This guide will customize your experience based on your environment.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/comanage/how-to-enable.md:26]:
```suggestion
> As a companion to this article, we recommend using the [Microsoft Intune Configuration Manager co-management setup guide](https://go.microsoft.com/fwlink/?linkid=2224782) when signed in to the Microsoft 365 admin center. This guide will customize your experience based on your environment.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @rededwin, kindly commit the minor suggested change. Many thanks.

**Comment by rededwin** [comment]
@gowdhamankarthikeyan PR has been copyedited and is ready for final review, could you kindly please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by gowdhamankarthikeyan — CHANGES_REQUESTED** [review]
Can you remove the "Microsoft Intune Configuration Manager" words and keep it simple as "co-management setup guide"?

File: memdocs/configmgr/comanage/how-to-enable.md [hunk 2]
```diff
@@ -22,6 +22,9 @@ The phrase **Pilot group** is used throughout the co-management feature and conf
 
 Make sure the co-management prerequisites are set up before you start this process. For more information, see [Prerequisites](overview.md#prerequisites).
 
+> [!TIP]
+> As a companion to this article, we recommend using the [Microsoft Intune Configuration Manager co-management setup guide](https://go.microsoft.com/fwlink/?linkid=2224782) when signed in to the Microsoft 365 admin center. This guide will customize your experience based on your environment.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/comanage/how-to-enable.md:26]:
```suggestion
> As a companion to this article, we recommend using the [co-management setup guide](https://go.microsoft.com/fwlink/?linkid=2224782) when signed in to the Microsoft 365 admin center. This guide will customize your experience based on your environment.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@rededwin, Kindly commit the suggested change as per @gowdhamankarthikeyan's request. Many thanks.

**Comment by rededwin** [comment]
@JohanFreelancer9 @gowdhamankarthikeyan please review and approve, thank you

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by rededwin** [comment]
@JohanFreelancer9 @gowdhamankarthikeyan when will this be published to external?

**Review by gowdhamankarthikeyan — APPROVED** [review]
Approved.

**Comment by gowdhamankarthikeyan** [comment]
#sign-off

**Comment by rededwin** [comment]
@gowdhamankarthikeyan when will this be published externally?  Thank you

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
@rededwin 
Hi,

This pull request was opened in the public repo. PMs and content developers should work in the private repo, as described in [the authoring guide](https://review.docs.microsoft.com/en-us/Office-Authoring-Guide/office-docs-authoring-overview?branch=main). 

We reviewed and merged this PR but encourage you to use the private repo for future contributions to Microsoft Learn.

The changes in this PR will sync with the private repo and create a new sync PR; if that PR is reviewed and merged before the 3:30 PubOps publish to live, it should be live shortly after that. 

Thanks!

**Comment by rededwin** [comment]
[like]  Edwin Der (HE/HIM/HIS) reacted to your message:
________________________________

**Pull request merged**